templates/compose/supabase.yaml

```diff
@@ -597,6 +604,46 @@ services:
       # Use named volume to persist pgsodium decryption key between restarts
       - supabase-db-config:/etc/postgresql-custom
 
+
+  supabase-supavisor:
+    image: supabase/supavisor:1.1.67
+    ports:
+      - 6543:6543
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "curl",
+          "-sSfL",
+          "--head",
+          "-o",
+          "/dev/null",
+          "-H",
+          "Authorization: Bearer ${SERVICE_SUPABASEANON_KEY}",
+          "http://127.0.0.1:4000/api/health",
+        ]
+      timeout: 5s
+      interval: 5s
+      retries: 3
+    environment:
+      - PORT=4000
+      - PROXY_PORT_SESSION=5452
+      - PROXY_PORT_TRANSACTION=6543
+      - DATABASE_URL=ecto://postgres:${SERVICE_PASSWORD_POSTGRES}@${POSTGRES_HOST:-supabase-db}:${POSTGRES_PORT:-5432}/${POSTGRES_DB:-postgres}
+      - CLUSTER_POSTGRES=true
+      - SECRET_KEY_BASE=12345678901234567890121234567890123456789012345678903212345678901234567890123456789032123456789012345678901234567890323456789032
```

Comment:
These probably shouldn't be hard coded

Reply:
What do you recommend? They need to specifically be this length

Reply:
Ideally there's a way to generate a random string either through docker-compose or coolify, I couldn't find one though.

Considering this should be unique to every user, and I don't imagine many people are changing the defaults while using these, I'd either wait for the functionality to be added into coolify, or force the user to enter the information.

You can get docker-compose to error if an environment variable is missing using `${VAR:?error}` so that would mean changing it to: `SECRET_KEY_BASE=${SECRET_KEY_BASE:?error}`.

Supabase is a fairly popular template, so people would need to be told that they need to put in a variable that contains a string that is X character long, but I don't know where a good place for the instruction to go would be.

Failing all that, maybe setting it to `SECRET_KEY_BASE=${SECRET_KEY_BASE}` and finding where the default values for Dashboard User and Dashboard Password are generated and ensure a new one is generated for the 2 variables that need them.

Reply:
Appears there is some functionality to do this: https://github.com/coollabsio/coolify/blob/e9158b7305bb9253fe75a0e76ef34af62f75199b/bootstrap/helpers/shared.php#L1969-L2046

Reply:
And the docs for how to use them: https://coolify.io/docs/knowledge-base/add-a-service#password

Reply:
Maybe this will work.

```yml
- SECRET_KEY_BASE=${SERVICE_BASE64_128_SUPAVISOR}
- VAULT_ENC_KEY=${SERVICE_PASSWORD_64_SUPAVISOR}
```

Doesn't actually need to be 128 characters I don't think, fly deployment docs suggest some secure value: https://supabase.github.io/supavisor/deployment/fly/

Might be worth setting them both to PASSWORD_64 if only for better naming.